src/lib/utils/shuffle.ts

```diff
@@ -1,14 +1,32 @@
-import Prando from 'prando';
-import { get } from 'svelte/store';
-import { gameProgress } from '$lib/store';
-
-export default (array: string[]) => {
-	const today = new Date();
-	const rng = new Prando(today.toDateString() + get(gameProgress));
-	for (let i = array.length - 1; i > 0; i--) {
-		const j = rng.nextInt(0, i + 1);
-		[array[i], array[j]] = [array[j], array[i]];
+import areArraysEqual from './areArraysEqual';
+import rng from './getRandomNumberGenerator';
+
+export const getShuffledWord = (letters: string[]): string[] => {
```

Comment:
I used the term "word" a lot in this file but now I'm realizing that these aren't really words we're scrambling, I should probably rename these